Automattic/jetpack
# Blocks Settings: Make Blocks toggleable (and searchable)

Follow-up from #23470, this adds "blocks" as a classic Jetpack module. This allows users to toggle off, if desired. It also allows us to surface the blocks when searching in settings. 

![image](https://user-images.githubusercontent.com/7129409/160158460-af891ff7-0534-4de4-b9e5-b63f0e0efb97.png)

Keeping this in progress until 23470 is merged and then will rebase. 

#### Changes proposed in this Pull Request:

- Mocks a bit of module data in the initial state to define some search terms.
- Use the mocked data structure to make the settings card searchable. 
- Blocks can now be disabled via this setting.

#### Jetpack product discussion


N/a

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






- We're going to want to **ensure that blocks are still on by default** after upgrading to the new version, and for brand new sites. 
- Search for the terms in settings, does the card show up? 
- Try deactivating/reactivating, test if it's actually doing expected.
